app/models/like.rb

```diff
@@ -1,4 +1,8 @@
 class Like < ApplicationRecord
-  belongs_to :user
-  belongs_to :post
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, counter_cache: :likes_counter
+
+  def increment_like_counter
+    post.increment!(:likes_counter)
+  end
```

Comment:
### **Optional**
- [ ] Please use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#increment_like_counter` method every time a new like has been saved to the database, then make the `increment_like_counter` method private.